src/interfaces/assistants_web/src/components/Shared/Text/Text.tsx

```diff
@@ -68,7 +68,7 @@ export const Text = <T extends AsElement>({
   ...rest
 }: TextProps<T>) => {
   const renderAs: AsElement = as ?? 'p';
-  const classes = cn(getStyleLevelClasses(styleAs ?? renderAs), className);
+  const classes = cn(getStyleLevelClasses(styleAs ?? renderAs), 'dark:text-marble-950', className);
```

Comment:
nit: as a fast follow, we should remove all the <Text> component which have added this class already